.env.example

```diff
@@ -1,3 +1,4 @@
 REACT_APP_AAD_CLIENT_ID=5cdcbb41-958a-43b6-baa1-bbafd80b4f70
-REACT_APP_AAD_TENANT_ID=513fb495-9a90-425b-a49a-bc6ebe2a429e
-REACT_APP_API_URL=http://localhost:8080/spring-api
\ No newline at end of file
+REACT_APP_AAD_TENANT_ID=common
```

Comment:
I believe this should still be: `513fb495-9a90-425b-a49a-bc6ebe2a429e` as the tenant ID.

We use `common` when the app registration is across tenants, i.e: you can sign in to MCA's Azure AD and any other AD.

The docs are here on this config: https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/initialization.md#optional-configure-authority

Reply:
Ahh - I think I had an old version of the .example with `common` in, will amend!